src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Since this is a logo link, it should typically point to the root of the site (e.g., `href="/"`), unless 'index.html' is a specific requirement.